iteration/iteration.go

```diff
@@ -69,6 +75,7 @@ type Repository interface {
 	CanStart(ctx context.Context, i *Iteration) (bool, error)
 	LoadMultiple(ctx context.Context, ids []uuid.UUID) ([]Iteration, error)
 	LoadChildren(ctx context.Context, parentIterationID uuid.UUID) ([]Iteration, error)
+	CalculateActiveIterationOnUpdate(ctx context.Context, itr *Iteration) (*bool, error)
```

Comment:
I changed the Implementation to remove those functions. The new implementation is in this commit -> 4052a1db091e0bc290ffd82768401ef164b875fb